packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -628,3 +694,54 @@ export const waitTextVisible = ({ context, text, isVisible = true }: { text: Pat
         }
     });
 };
+
+const tryToContinueOnInvalidPage: ContinueWithInvalidEntriesTest = async ({ context, text, currentPageUrl, nextPageUrl }) => {
+    const button = context.page.getByRole('button', { name: text });
+    await button.scrollIntoViewIfNeeded();
+    await button.click();
+    await expect(context.page.waitForURL(nextPageUrl, {timeout: 1000})).rejects.toThrow("Timeout")
```

Comment:
Is 1000ms enough to say that we did not go to the next page? Please add a FIXME above this line to validate this, or see if there's another condition. If we hit problems on slower machines, we'll know where to look at.